hack/import-restrictions.yaml

```diff
@@ -61,7 +61,7 @@
 - baseImportPath: "./vendor/k8s.io/client-go/tools/"
   excludeTests: true
   ignoredSubTrees:
-  - "./vendor/k8s.io/client-go/tools/bootstrap/token/api"
+  - "./vendor/k8s.io/bootstrap/token/api"
```

Comment:
this line can go.

But we need another similar block for the whole k8s.io/bootstrap directory.

Reply:
https://github.com/kubernetes/publishing-bot/pull/89

Reply:
https://github.com/kubernetes/kubernetes/pull/67356/commits/06a3a3bcd7f891d3074bcfd83e5215d99e068c11